package.json

```diff
@@ -15,6 +15,8 @@
     "@dqbd/tiktoken": "^1.0.2",
     "@tabler/icons-react": "^2.9.0",
     "eventsource-parser": "^0.1.0",
+    "formidable": "^2.1.1",
+    "fs": "^0.0.1-security",
```

Comment:
what is formidable and fs and where did they come from? curious why you would need new npm packages

Reply:
file stuff sending files as form in post request